wp-content/themes/redmond/web/scss/modules/_call-to-action.scss

```diff
@@ -11,6 +11,10 @@
     left: 0;
     color: $secondary-light-color;
     padding: rem-calc(20px) 0;
+    @include breakpoint(medium only) {
```

Comment:
remove the only. only means only on medium. instead we want to do just medium meaning medium and larger.

Generally we style for small and then use medium to make changes for larger screens and large for even larger. usually we don't need a lot for large.